app/routes/application.js

```diff
@@ -13,8 +13,8 @@ export default class ApplicationRoute extends Route {
       });
       const userData = await response.json();
       if (response.status === 200 && !userData.incompleteUserDetails) {
-        const firstName = userData.first_name || defaultName;
-        const profilePictureURL = userData.picture.url || defaultPicture;
+        const firstName = userData?.first_name || defaultName;
+        const profilePictureURL = userData?.picture.url || defaultPicture;
```

Comment:
Are you sure this will fix the problem?

Reply:
Was there any catch here :thinking: After this, I didn't get the error in console.